modules/administration-guide/partials/proc_backing-up-a-che-instance-to-the-internal-server-by-using-the-checlusterbackup-custom-object.adoc

```diff
@@ -0,0 +1,56 @@
+[id="backing-up-a-{prod-id-short}-instance-to-the-internal-server-by-using-the-checlusterbackup-object_{context}"]
+= Backing up a {prod-short} instance to the internal server by using the CheClusterBackup custom object
+
+When you create the custom object to back up a {prod-short} instance to the internal backup server, the {prod-short} Operator configures the internal backup server.
+
+WARNING: Configuring a backup server inside the same cluster, for example the internal backup server, is not the best practice.
```

Comment:
@mmorhun Something like this? If you have a better suggestion, let me know here.
```suggestion
WARNING: Configuring a backup server inside the same cluster, for example the internal backup server, is not the best practice because a cluster failure can be one of the scenarios when a backed up {prod-short} instance needs to be restored.
```

Reply:
Let's keep your version